components/04-views/registration/registration.hbs

```diff
@@ -0,0 +1,46 @@
+{{ render '@header' }}
+
+<main class="container registration">
+
+    <h1 class="heading registration__title">
```

Comment:
h1 and heading share the same code: 
(from components/01-globals/typography/heading/_heading.scss )
```
h1,
h2,
h3,
h4,
h5,
h6 {
    @extend .heading;
}
```
If so..is it necessary to use this `heading` class here ?

Reply:
no. thx. part of html structure was copied from other project not to write from scratch and forgot to check if it's necessary.